credentials/credentials.go

```diff
@@ -181,6 +181,6 @@ func List(helper Helper, writer io.Writer) error {
 
 // PrintVersion outputs the current version.
 func PrintVersion(writer io.Writer) error {
-	fmt.Fprintln(writer, Version)
+	fmt.Fprintln(writer, Package, Version, Revision)
```

Comment:
If we're gonna change this output, perhaps stick closer to the POSIX; https://www.gnu.org/prep/standards/html_node/_002d_002dversion.html#g_t_002d_002dversion


- first element; "name" (which could be `docker-credential-wincred`, or a "name" (`Docker Credential Helper for Pass`)
- Package name (should be overridable), in brackets (`(docker-credential-wincred <version>)`); perhaps that version could be the commit? 🤔.
- Version should be last

Reply:
Note that the POSIX also shows that license can be included (as second line, e.g.), additional lines are allowed, so we could go the `runc` path;

```bash
runc --version
runc version 1.1.2
commit: v1.1.2-0-ga916309
spec: 1.0.2-dev
go: go1.17.11
libseccomp: 2.5.1
```

Reply:
Made some changes:

```
$ make pass
$ ./bin/docker-credential-pass version
docker-credential-pass (github.com/docker/docker-credential-helpers) v0.6.4-36-g67c7cbd.m
```

Not necessary to display revision based on how versioning is handled but still fine to have it linked at build time.